### PR TITLE
Add sticky support in Query block

### PR DESCRIPTION
**Pull request by ntsekouras**
## Description
Part of: https://github.com/WordPress/gutenberg/issues/24934

This PR adds `sticky` support in the `Query` block. There are three options:
- Show all posts
- Include sticky
- Exclude sticky

Reference: https://developer.wordpress.org/rest-api/reference/posts/#arguments 

Another not related thing this PR changes, is some whitespaces differences that occurred during the regeneration of fixtures. The regeneration tool uses `spaces` and there were some handwritten with `tabs`.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by mariohamann** [comment]
As described over [here](https://github.com/WordPress/gutenberg/issues/25198#issuecomment-708222120) there are three options:

- `/wp-json/wp/v2/posts`  → show sticky and non-sticky posts
- `/wp-json/wp/v2/posts?sticky=true`  → show only sticky posts
- `/wp-json/wp/v2/posts?sticky=false`  → show only non-sticky posts

(Locally tested.)

The sticky option itself is a boolean – but using the option at all is another option, therefore we have 3 options and it’s definetely needed. If one wants to show “only sticky posts” in one Query Block, you need the option to exclude sticky posts in another Query Block to prevent duplicates.

**Comment by ntsekouras** [comment]
>The sticky option itself is a boolean – but using the option at all is another option, therefore we have 3 options

Hey @mariohamann - thanks for pointing this out! I pushed changes to handle all cases.

**Comment by scruffian** [comment]
I wonder if a better label for the first option would be "include sticky posts"?
Screenshot 2020-10-20 at 15 17 01

**Comment by mariohamann** [comment]
@scruffian The naming was my proposal, but I hadn't the label in mind when thinking about it (only thought about the options). I think you are absolutely right. What do you think about the following proposals to make it shorter and – in my opinion – more clear?

```
Sticky posts:
- Include
- Exclude
- Only
```
OR
```
Sticky posts:
- Include
- Only
- Exclude
```

**Comment by scruffian** [comment]
I like both. I'd put `Only` last as its the least likely to be used...

**Comment by mariohamann** [comment]
@ntsekouras 👍🏻 Thanks for relabeling once more!

File: packages/block-library/src/query-loop/edit.js [hunk 1]
```diff
@@ -65,6 +66,9 @@ export default function QueryLoopEdit( {
 			if ( exclude?.length ) {
 				query.exclude = exclude;
 			}
+			if ( sticky ) {
+				query.sticky = sticky === 'only';
```
Comment (mtias) [inline packages/block-library/src/query-loop/edit.js:70]: might need a comment here explaining what it does

**Review by mtias — APPROVED** [review]

**Review by aristath — APPROVED** [review]

**Pull request merged**